### PR TITLE
minor EN stat cleanup

### PR DESCRIPTION
**Pull request by SaveTheRbtz**

### CODE REVIEW

**Review by pattyshack — APPROVED** [review]

File: ledger/complete/ledger.go [hunk 1]
```diff
@@ -223,7 +223,7 @@ func (l *Ledger) Set(update *ledger.Update) (newState ledger.State, trieUpdate *
 	l.metrics.UpdateDuration(elapsed)
 
 	if len(trieUpdate.Paths) > 0 {
-		durationPerValue := time.Duration(elapsed.Nanoseconds()/int64(len(trieUpdate.Paths))) * time.Nanosecond
```
Comment (zhangchiqing) [inline ledger/complete/ledger.go:226]: can you explain why?

Comment (SaveTheRbtz) [reply ledger/complete/ledger.go:226]: time.Duration already gives the number in nanoseconds, so multiplying by time.Nanosecond (1) does not matter.

**Review by zhangchiqing — APPROVED** [review]

**Comment by SaveTheRbtz** [comment]
bors merge

**Comment by SaveTheRbtz** [comment]
bors retry

**Pull request merged**